server/src/assets/game/tableGenerator/Readme.md

```diff
@@ -0,0 +1,23 @@
+# Table Generator for SWTbahn Game
+
+This application generates the configuration json files for the swtbahn game client, which are required for the signal-symbol mapping and the route suggestion mechanics.
+Once entered the signals to symbol, the programm will automatically find a suitable route (which the least nodes) and store it for the game client.
+There is also the option to block single route ids within the blacklist.txt
+
+
+## Requirements
+* Python3
+* Configurationfiles of Bahn
+    * Interlocking table
+    * extra configuration (extra_config.yaml)
```

Comment:
should also list the "normal" config files, or are they not necessary?

Reply:
shouldnt be necessary, all data the script need from the configuration files of the model railway are provided by the interlocking table and the extras_config.yaml

![image](https://github.com/uniba-swt/swtbahn-cli/assets/48696584/9e9e7b14-146d-4704-88dc-92efc4101d5e)